project/src/test/test-world-data.gd

```diff
@@ -0,0 +1,4 @@
+extends GutTest
+
+func test_get_world_title() -> void:
```

Comment:
Expand this to test some other stuff, or remove the test.

Reply:
Slightly expanded this test to test the other titles, but overall I don't want to overdo it. This code is incredibly simple and boils down to loading data from a JSON file; past a point we're not really testing the code, we're just testing the JSON file.